### ISSUE TITLE
[Website]: Code preview no longer working

### ISSUE THREAD

**Issue by joshbruce** [18F/web-design-standards#1193]
## Issue type

* Website

## Description

1. Re-forked to create PR #1192 (for some reason I couldn't commit to origin staging...different matter).
1. Was validating working accordions.
1. Noticed code accordions not displayed.
1. Accordions working.

## Steps to reproduce the issue

1. Clone.
1. Serve.
1. Navigate to page with code preview.
1. No code accordion.
1. Open web inspector (see error message - attached).
1. Inspecting further demonstrated error not related js to files changed in PR #1192; therefore, believe issue is unrelated to that PR. (Related file: menu button and overlay.)

Note: I've had a similar issue happen when performing workarounds for Politespace. Current workaround seems to be: `var $ = window.jQuery = require('jquery');`

## Additional information [optional]

<img width="777" alt="screen shot 2016-05-25 at 6 11 09 pm" src="https://cloud.githubusercontent.com/assets/15252830/15557921/1ee60be6-22a4-11e6-89bb-886f5c98133e.png">

<img width="771" alt="screen shot 2016-05-25 at 6 12 26 pm" src="https://cloud.githubusercontent.com/assets/15252830/15557951/60bf83da-22a4-11e6-99f4-d75b0363100e.png">

**Comment by joshbruce** [comment]
Verified against serving default `staging` branch.

**Comment by maya** [comment]
Thanks @joshbruce! Re-pasting from my duplicate issue...

Errors:

```
styleguide.js:1 Uncaught ReferenceError: $ is not defined
uswds.min.js:3 Uncaught Error: Did you forget to define selectors in the aria-controls attribute? Check element undefined
```

From @el-mapache:
```
@maya I know that the uswds.min.js error is caused by the start.js script running on pages where the DOM element being jQuery selected isn't available (i.e. targeting a form control on the accordion page). I added the error message when I put in the PR for JS testing.

Since start.js runs on every page, what do you think the best way to deal with an element that isn't there? We could add a conditional check in that script that only initializes plugins when the appropriate element exists.
```

**Comment by rogeruiz** [comment]
This should be resolved with #1206 once it [auto-deploys to staging](https://circleci.com/gh/18F/web-design-standards/190). @joshbruce can you take a look. I believe we've squashed 💥 the bug 🐛

**Comment by maya** [comment]
I've tested this is working and fixed in #1206. Thanks again @joshbruce! 👍
